app/helpers/heatmap_helper.rb

```diff
@@ -27,7 +27,7 @@ module HeatmapHelper
   ].freeze
 
   # Samples and background are assumed here to be vieweable.
-  def self.sample_taxons_dict(params, samples, background_id)
+  def self.sample_taxons_dict(params, samples, background_id, client_filtering_enabled = false)
```

Comment:
I started to user keyword arguments in ruby. It's not a pattern we follow for all currently, but i think it is better than just positional with default, because it allows you to have multiple optional parameters without having to set them all. I have been starting to push for it for new method declarations. Wdyt?

`def self.sample_taxons_dict(params, samples, background_id, client_filtering_enabled: false)`

Reply:
Sounds good to me! I also like how it forces you to be more explicit, I think it makes it more readable/understandable at first glance.